pytest_selenium/pytest_selenium.py

```diff
@@ -194,8 +198,11 @@ def pytest_runtest_makereport(item, call):
                 _gather_html(item, report, driver, summary, extra)
             if 'logs' not in exclude:
                 _gather_logs(item, report, driver, summary, extra)
+            # gather debug from hook implementations
             item.config.hook.pytest_selenium_capture_debug(
                 item=item, report=report, extra=extra)
+    if driver is not None:
```

Comment:
Ah you are checking the driver twice. I was thinking it was only needing to be done once.

Reply:
Yeah, this seemed the least complicated approach.